app/components/view-epics/macro.test.js

```diff
@@ -0,0 +1,458 @@
+import request from 'supertest';
+import cheerio from 'cheerio';
+import { createTestHarness } from '../../testUtils/testHarness';
+
+const macroWrapper = `{% from './components/view-epics/macro.njk' import viewEpics %}
+                        {{ viewEpics(params) }}`;
+
+describe('view-epics', () => {
+  describe('must', () => {
+    it('should render the must epics tag if data provided', (done) => {
+      const context = {
+        params: {
+          dataTestId: 'some-data-identifier',
+          data: {
+            must: {
+              met: [
+                {
+                  id: 'C14E1',
+                  name: 'access prescribable items',
+                },
+              ],
+              'not-met': [
+                {
+                  id: 'C14E13',
+                  name: 'access Patient Record',
+                },
+              ],
+            },
+          },
+        },
+      };
+
+      const dummyApp = createTestHarness(macroWrapper, context);
+      request(dummyApp)
+        .get('/')
+        .then((res) => {
+          const $ = cheerio.load(res.text);
+          expect($('[data-test-id="some-data-identifier"] .must-epics .bc-c-tag').text().trim()).toEqual('Must Epics');
+
+          done();
+        });
+    });
+
+    it('should render the must data if provided', (done) => {
```

Comment:
please can you rename the tests to be more descriptive - e.g. renders viewEpicsLink component